js/admin/src/components/BazaarPage.js

```diff
@@ -37,46 +42,62 @@ export default class BazaarPage extends Component {
         ]);
     }
 
-    connectedHeader() {
+    header() {
+        var buttons = [].concat(this.requirementsButtons(), this.connectedButtons());
+
+        return m('div', {className: 'ButtonGroup'}, buttons);
+    }
+
+    /**
+     * Loads a list of buttons that give insight in the state of this installation.
+     * @returns {Array}
+     */
+    requirementsButtons() {
+        let memory_limit_met = app.data.settings['flagrow.bazaar.php.memory_limit-met'] || false;
```

Comment:
[Let is like a constant](http://wesbos.com/let-vs-const/) but only for the specific scope. I actually didn't know what the article explains before using it, but it makes a lot of sense :)